Templates/HttpTrigger-NodeJS/index.js

```diff
@@ -1,17 +1,17 @@
-﻿module.exports = function (context, req) {
-    context.log('Node.js HTTP trigger function processed a request. RequestUri=%s', req.originalUrl);
+﻿module.exports = function(context, req) {
+    context.log('Node.js HTTP trigger function processed a request. RequestUri:', req.originalUrl);
```

Comment:
It seems strange to pass multiple params with no format string. Not intuitive what it even means.